mesa_frames/concrete/pandas/agentset.py

```diff
@@ -0,0 +1,439 @@
+from collections.abc import Callable, Collection, Iterable, Iterator, Sequence
+from typing import TYPE_CHECKING
+
+import geopandas as gpd
+import pandas as pd
+import polars as pl
+from typing_extensions import Any, Self, overload
+
+from mesa_frames.abstract.agents import AgentSetDF
+from mesa_frames.concrete.pandas.mixin import PandasMixin
+from mesa_frames.concrete.polars.agentset import AgentSetPolars
+from mesa_frames.types_ import PandasIdsLike, PandasMaskLike
+
+if TYPE_CHECKING:
+    pass
+
+
+class AgentSetPandas(AgentSetDF, PandasMixin):
```

Comment:
What is the difference between `AgentSetPandas` here and the one in agentset_pandas.py?

Reply:
Sorry I started refactoring but I discard changes and OneDrive made a mess with file history. Nothing changes, just moved agentset to it's library specific folder.